scripts/uninstall_module.py

```diff
@@ -10,6 +10,12 @@
 import subprocess
 import sys
 
+# With Python 3.13 the subprocess module uses `posix_spawn()` function
+# which requires loading the `signal` module:
+#     https://docs.python.org/3/whatsnew/3.13.html#subprocess
+# flake8: noqa: F401
```

Comment:
From this comment, it is unclear why this module should be imported here.

Reply:
I've updated the comment to clarify the reason why this module should be imported here.